Xamarin.Forms.Platform.Android/CollectionView/ItemsViewRenderer.cs

```diff
@@ -483,5 +492,77 @@ internal void UpdateEmptyViewVisibility()
 				SetLayoutManager(SelectLayoutManager(_layout));
 			}
 		}
+
+		public class CustomGridLayoutManager : GridLayoutManager
+		{
+			public CustomGridLayoutManager(Context context, int spanCount, int orientation, bool reverseLayout) : base(context, spanCount, orientation, reverseLayout)
+			{
+
+			}
+		}
+
+		public class CustomLinearLayoutManager : LinearLayoutManager
```

Comment:
Extended the default layout managers because Android calculates horizontal/vertical scroll offsets by first calculating an average row height instead of looking at every row height. Unlike iOS, `RecyclerView` does not report actual offsets. See [here](https://stackoverflow.com/a/32863117/246392).